be/src/runtime/fragment_mgr.cpp

```diff
@@ -1609,4 +1620,55 @@ void FragmentMgr::get_runtime_query_info(std::vector<WorkloadQueryInfo>* query_i
     }
 }
 
+Status FragmentMgr::get_realtime_exec_status(const TUniqueId& query_id,
+                                             std::shared_ptr<TReportExecStatusParams> exes_status) {
```

Comment:
std::shared_ptr<TReportExecStatusParams>* exes_status
OR the value will not be returned.
And you could also use std::shared_ptr<TReportExecStatusParams>&, but I do not think we could use it like this. Because in doris, we use pointer as return values, and reference as input parameters.

Reply:
ok i will use raw pointer.